packages/react-codemirror/src/lang-cypher/signatureHelp.ts

```diff
@@ -20,14 +18,6 @@ function getTriggerCharacter(query: string, caretPosition: number) {
   return triggerCharacter;
 }
 
-function getDocString(result: string | MarkupContent): string {
-  if (MarkupContent.is(result)) {
-    result.value;
-  } else {
-    return result;
-  }
-}
-
```

Comment:
I've moved this to a common utils file since it's needed for the auto-completions as well